components/x-audio/src/components/handle-swipe-down.js

```diff
@@ -0,0 +1,21 @@
+function setPositionY (target, posY) {
+	target.style.transform = `translate3d(0, ${posY}px, 0)`;
+}
+
+export default (event, onSwipeEnd, expandedPlayerRef) => {
+
+	const DISPLAY_EXPANDED_THRESHOLD = expandedPlayerRef.offsetHeight / 2;
+	const posY = event.deltaY;
+
+	setPositionY(expandedPlayerRef, posY);
+
+	// SWIPE ENDED
+	if (event.isFinal) {
+		if (posY >= DISPLAY_EXPANDED_THRESHOLD) {
+			onSwipeEnd();
```

Comment:
One tiny thing, do we want to `return` here? If I understand it correctly there could be a very short delay before the `onMinimise` action does its thing, which means the player could possibly flick back to the top (because of line 18)?

Reply:
📝 There is a reason, I couldn't `return` there. It needs to run `setPositionY(expandedPlayerRef, 0);` after `onSwipeEnd()`. I'm going to add comments to make the situation clear.